src-tools/netmodel/netmodel.cpp

```diff
@@ -1273,7 +1280,7 @@ void asiotest_udpserv(std::vector<std::string> options) {
 	// stop / show stats
 	_goal("The stop (and stats) thread"); // exit flag --> ios.stop()
 	std::thread thread_stop(
-		[&ios_general,&ios_wire,&ios_tuntap, &ios_general_work, &ios_wire_work, &ios_tuntap_work, &welds, &welds_mutex, &cfg_run_timeout] {
+        [&ios_general,&ios_wire,&ios_tuntap, &ios_general_work, &ios_wire_work, &ios_tuntap_work, &welds, &welds_mutex, &cfg_run_timeout,&cfg_stats_format] {
```

Comment:
perhaps better stick to passing to lambda as copy to have 1 less thing that could go wrong (regarding lifetime)